src/main/java/com/checkmarx/flow/config/ADOProperties.java

```diff
@@ -8,6 +10,8 @@
 @ConfigurationProperties(prefix = "azure")
 @Validated
 public class ADOProperties extends RepoProperties{
+
+    private boolean deleteCxProject = false;
```

Comment:
If you are using Lombok Getter & Setter you can add the @Getter & Setter annotations and neglects the get & set methods

Reply:
removed lombok imports, I wanted keep existing behavior of that class